src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './errors'
 export * from './middleware'
 export * from './types'
 export * from './validators'
+export { default as defaultReporter } from './reporter'
```

Comment:
This comment is not related to the PR directly, but might help you in the future a bit. I stopped using default exports for anything except for React components and my DX became noticeably better.

When re-exporting files from a directly, I can now always do:

```ts
export * from "./a";
export * from "./b";
export * from "./d";
```

If there is a file that exports both ‘private’ and ‘public’ things, I split it into two and don’t re-export stuff from the ‘private’ file. In the example above, it would be `"./c"`.


Here’s a blog post on this topic:
https://basarat.gitbook.io/typescript/main-1/defaultisbad